src/components/ColumnWidth.sc.js

```diff
@@ -48,6 +48,15 @@ const ContentOnRow = styled.div`
     margin: 0.2em;
   }
 `;
+const ContentOnRowModal = styled.div`
+  display: flex;
+  flex-direction: row;
+  align-items: center;
```

Comment:
Actually, this is not needed anymore as it was part of styling for the Modal solution.

We can pass a parameter, but I think it would have to be overriden over style, not sure you can pass a value into a constant. Maybe you can make a function that return a styled.div?